### DISCUSSION TITLE
android studio plugin no longer compatible

### DISCUSSION THREAD

**Discussion by mujehoxe** [Genymobile/feedback#68] [Genymotion Desktop] [answered]
![Screenshot from 2023-08-26 14-02-02](https://github.com/Genymobile/feedback/assets/52047683/f13966f1-623f-4fb4-bebe-3be724b0bdf2)

**Comment by jmaneyrol69** [comment]
Hi @mujehoxe,
We are currently working on an update for this plugin, and it should be released very soon.
Make sure to checkout our blog at https://www.genymotion.com/blog/ to get the latest news.
Thank you.

**Comment by agavignet** [comment, marked as answer]
Hi @mujehoxe, a new version of the plugin has been released and should fix your issue, you can read the release note [here](https://www.genymotion.com/blog/release-note/genymotion-plugins-2-0-0/)
Thanks for your patience
